inc/hooks.php

```diff
@@ -133,3 +133,130 @@ function _s_custom_mime_types( $mimes ) {
 // Create shortcode for SVG.
 // Usage [svg icon="facebook-square" title="facebook" desc="like us on facebook" fill="#000000" height="20px" width="20px"].
 add_shortcode( 'svg', '_s_display_svg' );
+
+/**
+ * Adds OG tags to the head for better social sharing.
+ *
+ * @return string Just an empty string if Yoast is found.
+ * @author Corey Collins
+ */
+function _s_add_og_tags() {
+
+	// Bail if Yoast is installed, since it will handle things.
+	if ( class_exists( 'WPSEO_Options' ) ) {
+		return '';
+	}
+
+	global $post;
+
+	// Get the post content.
+	$post_content = $post ? $post->post_content : '';
+
+	// Strip all tags from the post content we just grabbed.
+	$default_content = wp_strip_all_tags( strip_shortcodes( $post_content ) );
+
+	// Set our default title.
+	$default_title = get_bloginfo( 'name' );
+
+	// Set our default URL.
+	$default_url = get_permalink();
+
+	// Set our base description.
+	$default_base_description = get_bloginfo( 'description' ) ? get_bloginfo( 'description' ) : __( 'Visit our website to learn more.', '_s' );
+
+	// Set the card type.
+	$default_type = 'article';
+
+	// Get our custom logo URL. We'll use this on archives and when no featured image is found.
+	$logo_id    = get_theme_mod( 'custom_logo' );
+	$logo_image = $logo_id ? wp_get_attachment_image_src( $logo_id, 'full' ) : '';
```

Comment:
@coreymcollins Sorry to be so picky, but ternary operators should have a `()`. It's not explicitly mentioned in our [PHP Code Standards](https://make.wdslab.com/docs/engineering/php-coding-standards/), but the ternary on this page does use a parenthesis. I would consider this a code smell issue, and not a pass/fail blocker.